aws-redshiftserverless-workgroup/src/main/java/software/amazon/redshiftserverless/workgroup/BaseHandlerStd.java

```diff
@@ -23,12 +36,19 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
     public static final String BUSY_WORKGROUP_RETRY_EXCEPTION_MESSAGE =
             "There is an operation running on the existing workgroup";
 
+    protected static final String NAMESPACE_STATUS_AVAILABLE = "available";
```

Comment:
Can't we use NamespaceStatus enum instead?

Reply:
This is not being used will remove this.